src/functionalTest/resources/application.conf

```diff
@@ -1,3 +1,4 @@
+source-url=${TEST_URL}
```

Comment:
'source' seems weird here imho.
Maybe one of the below?
- `app-url`
- `service-url`
- `blob-router-url`